python/fledge/common/logger.py

```diff
@@ -23,12 +23,18 @@
 
 """
 CONSOLE = 1
-"""Send log entries to STDOUT"""
+"""Send log entries to STDERR"""
 
+FLEDGE_DESTINATION_ENV='FLEDGE_LOG_DEVICE'
```

Comment:
unused?

Reply:
the ENV is used -- it's an env variable that's queried to see if there's a default set when starting a new process, this allows the process to load the module, determine which env variable to check, then reset the default if set. We could also have the logging module check the ENV and reset the default itself when it loads? (I chose the option of letting clients do it as a little less invasive, but, probably it makes more sense for the module to do it, itself?)

Reply:
To set as an env variable, I think os.environ['FLEDGE_LOGS_DESTINATION'] = Should be used. And better to add some docstring comment to explain the need and usage.


> FLEDGE_DESTINATION_ENV better named as FLEDGE_LOGS_DESTINATION_ENV or just FLEDGE_LOGS_DESTINATION.

Reply:
OK, sounds good, will change.